### PR TITLE
Adjust horizontal footer logos style

### PR DESCRIPTION
**Pull request by albinazs**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1684897205)
(#310  re-created directly in the repo after getting the write access)

### Description of Changes Made

- Updated Employee owned logo in CMS (larger size that will scale) - currently on torchbox.com
- Updated `fill` dimensions to accommodate for horizontal logos and related CSS

### How to Test

Check the homepage on the local build. The result was okayed by Lily V.

### Screenshots

<details>
  <summary>Expand to see more</summary>

image

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [ ] Updated field spec (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Light and dark mode

- [ ] I have tested the changes in both light and dark mode
- [x] The change is not relevant to dark and light mode

#### Accessibility

- [ ] Automated WCAG 2.1 tests pass
- [ ] HTML validation passes
- [ ] Manual WCAG 2.1 tests completed
- [ ] I have tested in a screen reader
- [ ] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [x] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [ ] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [x] Changes are not relevant the pattern library

### CODE REVIEW

**Comment by helenb** [comment]
Thank you for recreating this pull request! Appreciate you didn't have access before.

I'm in the process of testing, and I think the logos are now too large. Here's a screenshot from the figma design:

Screenshot 2024-11-05 at 14 29 06

I am trying to have a play myself to see what crop might work, but have come across with a peculiar issue with the max-resized not working as expected when using png. I have raised a query about this in the wagtail slack channel.

**Comment by helenb** [comment]
> Thank you for recreating this pull request! Appreciate you didn't have access before.
> 
> I'm in the process of testing, and I think the logos are now too large. Here's a screenshot from the figma design:
> 
> Screenshot 2024-11-05 at 14 29 06
> 
> I am trying to have a play myself to see what crop might work, but have come across with a peculiar issue with the max-resized not working as expected when using png. I have raised a query about this in the wagtail slack channel.

That aside, I think something like this code could work - you might need to tweak the exact crop values:

```
{% if logo.image.width > logo.image.height %}
                                                    {% image logo.image max-200x100 format-webp loading="lazy" class="footer__logo" alt=logo.alt_text|default:logo.image.title %}
                                                {% else %}
                                                    {% image logo.image max-100x100 format-webp loading="lazy" class="footer__logo" alt=logo.alt_text|default:logo.image.title %}
                                                {% endif %}
```

**Review by helenb — COMMENTED** [review]
Thanks @albinazs - it would be great if you could change the crop so the images aren't too large, and adjust the crop if the image is landscape - see my code snippet for that.

**Comment by albinazs** [comment]
thank you a lot, @helenb!
this is committed now

**Comment by albinazs** [comment]
Final look:

image

File: tbx/static_src/sass/components/_footer.scss [hunk 1]
```diff
@@ -85,11 +85,14 @@
...
     &__logo {
         filter: grayscale(1) invert(1) brightness(0.52);
-        max-width: 70px;
 
         @include high-contrast-light-mode() {
             filter: none;
         }
+
+        &--narrow {
+            max-width: 70px;
```
Comment (helenb) [inline tbx/static_src/sass/components/_footer.scss:94]: One final tweak I think would be nice - at the moment the EO logo now looks very big at the larger screen sizes - maybe a max-width of about 150px for the landscape logos would be good?

**Review by helenb — APPROVED** [review]
Thanks @albinazs I've approved but there's one more tweak I think would be good if you have time.

File: tbx/static_src/sass/components/_footer.scss (hunk 1 above)
Comment (albinazs) [reply tbx/static_src/sass/components/_footer.scss:95]: Sure, added

**Pull request merged**